### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by Digitalone1** [wwmm/easyeffects#1591] [General]
@wwmm do you think the code is ready for a new stable release? The last one was quite two months ago and lots of changes and fixes have been made.

**Comment by wwmm** [comment]
I was thinking about this. It is probably time to do a new one. Do you need time for translations updates?

**Comment by Digitalone1** [comment]
No, Italian 100%, but I don't remember if translation templates of the changelog are fully updated to weblate.

**Reply by wwmm** [reply]
In this case it is fine to make a release today. Six days ago I updated the translation template and there was no change to the window strings since then.
